docs/tools/hardhat/hardhat-zksync-vyper.md

```diff
@@ -77,4 +120,38 @@ yarn hardhat compile
 @tab npm
 ```bash
 npx hardhat compile
+```
+:::
+
+### 4. Deploy Your Contract
+
+To deploy the `Greeter.vy` contract, initially update the `use-greeter.ts` script, supplied by the CLI in the `deploy/` directory. 
+
+Alter this line:
+```
+// Load contract artifact. Ensure to compile first!
+import * as ContractArtifact from "../artifacts-zk/contracts/Greeter.Sol/Greeter.json";
```

Comment:
```suggestion
import * as ContractArtifact from "../artifacts-zk/contracts/Greeter.sol/Greeter.json";
```

Reply:
is it really a capital S?

Reply:
Nope my mistake.

Reply:
i only asked because it was so consistent ... :))))